tests/test_rise.py

```diff
@@ -15,19 +15,20 @@ class RiseOnImages(TestCase):
     def test_rise_function(self):
         # shape is batch, y, x, channel
```

Comment:
Remove comment and ideally add the y and x labels to the `axes_labels` as well. They may not actually be useful in this case (that you could note in a comment), but it's still good practice to have descriptive data, I think.

Reply:
Agreed